boards/aithinker/ai_m61_32s_kit/ai_m61_32s_kit-pinctrl.dtsi

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	i2c0_default: i2c0_default {
+		group1 {
+			pinmux = <GPIO0_I2C0_SCL>,
+				 <GPIO1_I2C0_SDA>;
+			bias-pull-up;
+			input-schmitt-enable;
+		};
+	};
+
+	spi0_default: spi0_default {
+		group1 {
+			pinmux = <GPIO27_SPI_MOSI>,
+				 <GPIO29_SPI_SCLK>,
+				 <GPIO30_SPI_MISO>;
```

Comment:
Aren't GPIO 27 and GPIO 29 used by LEDs (see below)?
```dts
		white_led: led_3 {
			gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
			label = "White - LED3";
		};

		warmwhite_led: led_4 {
			gpios = <&gpio0 27 GPIO_ACTIVE_HIGH>;
			label = "Warm White - LED4";
		};
```

Reply:
It looks like `GPIO26_SPI_MISO` and `GPIO25_SPI_SCLK` would work, they're nearby ([doc](https://en.ai-thinker.com/Uploads/file/20230808/20230808020536_36304.pdf#page=11)):

![screenshot_2025_1203_2304_00](https://github.com/user-attachments/assets/5de96722-fd9d-406c-976e-2548b346d307)